src/cluster.c

```diff
@@ -5591,6 +5565,41 @@ NULL
                 (retval == C_OK) ? "BUMPED" : "STILL",
                 (unsigned long long) myself->configEpoch);
         addReplySds(c,reply);
+    } else if (!strcasecmp(c->argv[1]->ptr,"slot-stats")) {
+        unsigned char *slots = zmalloc(CLUSTER_SLOTS);
+        memset(slots, 0, CLUSTER_SLOTS);
+        if (c->argc == 2) {
+            /* CLUSTER SLOT-STATS */
+            markAssignedSlots(slots);
+            addReplySlotStats(c, slots);
+        } else if (c->argc >= 3 && !strcasecmp(c->argv[2]->ptr,"slots")) {
+            /* CLUSTER SLOT-STATS SLOTS slot [slot ...]*/
```

Comment:
My vote is still to kill the individual slots for now, we can always add it later but it simplifies the API.

Reply:
Looks like there's no further concerns.
Will have the `SLOTS` removed in the next revision.